modules/sandboxed-containers-viewing-workloads-from-web-console.adoc

```diff
@@ -3,5 +3,23 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-viewing-workloads-from-web-console_{context}"]
+= Viewing {sandboxed-containers-first} workloads from the web console
 
-= Viewing sandboxed containers workloads from the web console
+{sandboxed-containers-first} based workloads look and feel the same as normal workloads when viewed in the web console. The only difference between the two is the `runtimeClassName`. `runtimeClassName` is what decides the runtime used for workloads. In this context, the runtime enabled by {sandboxed-containers-first}-based is `kata`. You can view the `runtimeClass` that the pods for your workloads use.
```

Comment:
How does `{sandboxed-containers-first}-based` render in a preview? That is probably the correct punctuation, assuming it looks OK.

Reply:
It renders the approved product name which is OpenShift sandboxed containers